snapshot/snapshot.go

```diff
@@ -343,7 +343,11 @@ func (o *snapshotter) Mounts(ctx context.Context, key string) ([]mount.Mount, er
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to get active mount")
 	}
-	return o.mounts(ctx, s, key)
+	mounts, err := o.mounts(ctx, s, key)
+	if err == nil {
+		log.G(ctx).Info("SOCI mount successfully returned")
```

Comment:
should we print which layer is mounted successfully? (`s.ID`)

Reply:
As in the mount command? This will simply return the mount command with the upperdir and workdir set to the active snapshot.

Reply:
Or did you mean the active snapshot digest?

Reply:
I mean in the underlying `o.mounts` we log layer digest in the error message. should we also log the layer digest in this info line?

https://github.com/awslabs/soci-snapshotter/blob/eabacb9e7fd8c27fa27895aefaeaaaa9826ee831/snapshot/snapshot.go#LL627C6-L627C6

Reply:
I think that makes sense for more verbosity.